### ISSUE TITLE
app is unusable on macos 15.*

### ISSUE THREAD

**Issue by psebastian** [MindMacApp/MindMac#9]
It crashes and crashes and crashed. Deeply uninstalled the app, reinstalled it and same story.

After one in 3 messages sent the app crashes.

**Comment by lordnikkon** [comment]
+1

**Comment by RyuX51** [comment]
+1000

Really, please address the freezes with the highest priority, it renders the app pretty much unusable in macOS 15.

**Comment by SOY75** [comment]
Yes please, we need a fix, I have too many crashes especially with OpenRouter / o1. Thanks

**Comment by RyuX51** [comment]
Do my fellow sufferers: You can enable streaming now for o1 models, which decreases the chance of freezing by a lot, but it still happens.

**Comment by Holot** [comment]
+1

**Comment by tarnoff** [comment]
I'm having the same issue - extremely frustrating, as this has been going on for months now (since the beta version of MacOS Sequoia). At this point, it's hanging immediately upon startup, so it's completely unusable. I'm going to try completely uninstalling it and all prefs, etc. Has anyone found any workarounds?

**Comment by Holot** [comment]
No, even complete clean uninstall is not working

**Comment by augustofretes** [comment]
+1

**Comment by blankclemens** [comment]
+1

**Comment by shkicaz** [comment]
same here, had to make a shortcut to force quit the app and re-open it just to not go insane of doing the same cycle multiple times in a row until I get it to work for one response..

got this response from the dev 3 months ago

> Thank you for reaching out. We appreciate you taking the time to report this issue. It appears to be related to macOS 15, as other applications have experienced similar problems for some users. You can find more information and discussions on this topic https://www.reddit.com/r/MacOS/comments/1fihlge/macos_15_sequoia_bugs_and_issues_megathread. Our team has been actively investigating the matter, and we have been able to reproduce the issue on certain machines, while others continue to run MindMac without any problems. We are currently working on a workaround solution to address this issue, while also awaiting updates from Apple that may resolve it. Please stay tuned for further updates, and we apologize for any inconvenience caused by this situation.

**Comment by bdemasie** [comment]
Oh no! I upgraded to Sequoia yesterday and MindMac has been unusable since. Hoping for a fix soon.

**Comment by lenasi** [comment]
Same here—constant freezing. Any updates? In its current state, MindMac is unusable.

**Comment by sejmann** [comment]
Yeah, I kinda hate that I paid money for this abandonware. At the same time, I hope the developer is okay. In any case, it's not nearly as use-friendly to setup but I'm loving [Open WebUI](https://github.com/open-webui). I just use Safari's site-specific browser feature "File>Add to Dock" to launch it in a window. It's updated very frequently.

**Comment by borjarobert** [comment]
I have the same issue. MindMac is completely unusable in its current state

**Comment by ypanasiuk** [comment]
+1. 15.4 - MindMac freezes all the time

**Comment by RyuX51** [comment]
Guys, are you really **still** waiting for the author to fix this? Just switch to a better maintained alternative. After evaluating the other products, I switched to [Bolt AI](https://boltai.com/) and never looked back. It's in active development, packed with features and the author actually responds if you send him an email. And it **never** crashes. ;)

**Comment by bdemasie** [comment]
I've moved on to Msty

**Comment by ypanasiuk** [comment]
Impossible to use
